modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -536,6 +537,44 @@ public ObjectEntry getByExternalReferenceCodeByVersion(
 			_objectDefinition, null, version);
 	}
 
+	@Override
+	public Comment getByExternalReferenceCodeComment(
+			String externalReferenceCode, String commentExternalReferenceCode)
+		throws Exception {
+
+		if (!_objectDefinition.isEnableComments() ||
+			!FeatureFlagManagerUtil.isEnabled(
+				_objectDefinition.getCompanyId(), "LPD-69419")) {
+
+			throw new UnsupportedOperationException();
+		}
+
+		ObjectEntry objectEntry = _getObjectEntry(externalReferenceCode, null);
+
+		com.liferay.portal.kernel.comment.Comment serviceBuilderComment =
+			_fetchComment(
+				ObjectEntry.class.getName(), objectEntry.getId(),
```

Comment:
We should use `_objectDefinition.getClassName()` here. Please replace in all places.